packages/opentelemetry-operations-exporter-trace/README.md

```diff
@@ -0,0 +1,30 @@
+# OpenTelemetry Stackdriver Trace Exporter
+
+OpenTelemetry Stackdriver Trace Exporter allows the user to send collected traces to Stackdriver.
+
+[Stackdriver Trace](https://cloud.google.com/trace) is a distributed tracing system. It helps gather timing data needed to troubleshoot latency problems in microservice architectures. It manages both the collection and lookup of this data.
+
+## Setup
+
+Stackdriver Trace is a managed service provided by Google Cloud Platform.
+
+## Installation
+
+```bash
+npm install --save @google-cloud/opentelemetry-operations-exporter-trace
```

Comment:
What's the reason for the `operations` in the name? I would have thought that `@google-cloud/opentelemetry-exporter-trace` is a unique enough name on its own (I have not been so close to the OpenTelemetry JS repo so maybe there was a whole discussion that lead to that, but seems like just an extra word to me that people will have to have in their source code every time they import this)

Reply:
I am checking with stackdriver leads on the package name.